src/app/dataset-block/metadata-block/components/event-details/components/add-push-source-event/add-push-source-event.source.ts

```diff
@@ -0,0 +1,373 @@
+import { SetPollingSourceToolipsTexts } from "src/app/common/tooltips/tooltips.text";
+import { EventRowDescriptorsByField } from "../../dynamic-events/dynamic-events.model";
+import { SimplePropertyComponent } from "../common/simple-property/simple-property.component";
+import { AddPushSourceToolipsTexts } from "src/app/common/tooltips/add-push-source.text";
+import { StepTypePropertyComponent } from "../common/step-type-property/step-type-property.component";
+import { SchemaPropertyComponent } from "../common/schema-property/schema-property.component";
+import { SeparatorPropertyComponent } from "../common/separator-property/separator-property.component";
+import { MergeStrategyPropertyComponent } from "../common/merge-strategy-property/merge-strategy-property.component";
+import { EnginePropertyComponent } from "../common/engine-property/engine-property.component";
+import { SqlQueryViewerComponent } from "../common/sql-query-viewer/sql-query-viewer.component";
+import { CardsPropertyComponent } from "../common/cards-property/cards-property.component";
+
+export const ADD_PUSH_SOURCE_DESCRIPTORS: EventRowDescriptorsByField = {
+    "AddPushSource.string.sourceName": {
+        label: "Source name:",
+        tooltip: AddPushSourceToolipsTexts.SOURCE_NAME,
+        presentationComponent: SimplePropertyComponent,
+        separateRowForValue: false,
+        dataTestId: "add-push-source-name",
+    },
+
+    "AddPushSource.ReadStepCsv.__typename": {
+        label: "Type:",
+        tooltip: SetPollingSourceToolipsTexts.READ_CSV,
+        presentationComponent: StepTypePropertyComponent,
+        separateRowForValue: false,
+        dataTestId: "add-push-source-ReadStepCsv-__typename",
```

Comment:
For identical properties, if the only difference is prefix in `dataTestId`, we should have a mechanism to have a single definition with prefix given as a parameter.